plugins/transport-grpc/src/internalClusterTest/java/org/opensearch/transport/grpc/OpenSearchGrpcIntegTestCase.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.transport.grpc;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+import java.util.Collection;
+import java.util.List;
+
+public abstract class OpenSearchGrpcIntegTestCase extends OpenSearchIntegTestCase {
```

Comment:
With just the minimal framework here there's not really anything to integration test. Let's go ahead and remove this and the `apply plugin: 'opensearch.internal-cluster-test'` line from build.gradle because otherwise the `internalClusterTest` target will fail if there are no tests to execute.